main.py

```diff
@@ -0,0 +1,10 @@
+from car.models import Car
+from car.serializers import CarSerializer
+
+
+def serialize_car_object(car: Car) -> bytes:
+    pass
+
+
+def deserialize_car_object(json: bytes) -> CarSerializer:
+    pass
```

Comment:
Shouldn't it be the Car at the end?
Like Car -> bytes it's serialization, and bytes -> Car it's deserialization?

Reply:
Sounds logical, will fix